src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -148,7 +148,16 @@ export const selectPendingAppointments = createSelector(
 export const selectPastAppointments = createSelector(
   state => state.appointments.past,
   past => {
-    return past?.filter(isValidPastAppointment).sort(sortByDateDescending);
+    if (!past) {
```

Comment:
I think your changes in here will conflict with the current past appointments tab, so you might need to create a copy of this selector.

I also added some guidance I got from Peter around EC requests, some of which applies to the past appointments page: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/18164#issuecomment-764764507

Reply:
Per our slack discussion, not doing at thiis time.

Reply:
You don't need to do the express care part, but this is still I believe causing issues between the old homepage and new homepage. You still probably need to copy this selector, because `groupAppointmentsByMonth` will break the old page.